src/config/database.ts

```diff
@@ -1,15 +1,22 @@
 import { createConnection } from "typeorm";
 import { User } from "../api";
 
+interface DatabaseProps {
+  username: string;
+  password: string;
+  databaseName: string;
+  port: number;
+}
+
 export class Database {
-  static async config(): Promise<void> {
+  static async config({ port, username, password, databaseName }: DatabaseProps): Promise<void> {
     await createConnection({
       type: "postgres",
       host: "localhost",
-      port: 5432,
-      username: "onboard",
-      password: "onboard",
-      database: "onboard",
+      port: Number(port),
```

Comment:
Outra opção legal tbm pra transformar em número é o `+`:

```suggestion
      port: +port,
```

Reply:
Nice. Vou aplicar nos outros nos outros PRs também